spk/mitmproxy/src/installer.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+
+# Package
+PACKAGE="mitmproxy"
+DNAME="mitmproxy"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="mitmproxy"
+GROUP="users"
```

Comment:
@ymartin59 Just to check: are you sure the `nobody` group is available on a clean DSM6 environment, and not that it was created by one of our packages you have installed?
As for the `subliminal` user, yes, that's expected if you installed the user while on DSM5. It's not a DSM6 package...none of our packages are.